RequestManagementSystem/Client/ReqClient.py

```diff
@@ -330,27 +342,25 @@ def finalizeRequest(self, requestID, jobID, useCertificates=True):
       else:
         self.log.error("finalizeRequest: Failed to get request digest for %s: %s" % (requestID,
                                                                                      digest["Message"]))
-      stateUpdate = None
-      if jobStatus == 'Completed':
+      if jobStatus == JobStatus.COMPLETED:
         # What to do? Depends on what we have in the minorStatus
         if jobMinorStatus == "Pending Requests":
           newJobStatus = 'Done'
-
         elif jobMinorStatus == "Application Finished With Errors":
           newJobStatus = 'Failed'
 
-      if newJobStatus != jobStatus:
+      if newJobStatus:
         self.log.info("finalizeRequest: Updating job status for %d to %s/Requests done" % (jobID, newJobStatus))
-        stateUpdate = stateServer.setJobStatus(jobID, newJobStatus, "Requests done", "")
       else:
         self.log.info(
-            "finalizeRequest: Updating job minor status for %d to Requests done (status is %s)" %
+            "finalizeRequest: Updating job minor status for %d to Requests done (current status is %s)" %
```

Comment:
```suggestion
            "finalizeRequest: Updating job minor status", "for %d to Requests done (current status is %s)" %
```
(split fixed from variable part)

Reply:
OK